internal/users/cache/delete.go

```diff
@@ -118,17 +119,17 @@ func deleteOrphanedUsers(db *bbolt.DB) error {
 		return buckets[userByIDBucketName].ForEach(func(k, v []byte) error {
 			var user UserDB
 			if err := json.Unmarshal(v, &user); err != nil {
-				slog.Warn(fmt.Sprintf("Error loading user record {%s: %s}: %v", string(k), string(v), err))
+				log.Warningf(context.TODO(), "Error loading user record {%s: %s}: %v", string(k), string(v), err)
```

Comment:
Do not use `string()` "casting", since using `%s` will make this happen only if required (and so only if the log level is enabled)

Reply:
That line was changed by a regex search and replace. I will remove the `string()` in another commit.

Reply:
done